### ISSUE TITLE
[Accessibility- Information presentation C 10.7] News_details-News application- Visual indication of focus

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3019]
**Environment:** Tribe

**Step of reproduction:**
Access to any article and News application, then enable the focus of items by tabbing or by using pointing device

**Current behavior:**
A lot of interactive elements on the platform currently have a focus style defined but not visible enough:

- In article:

- [ ] The back button

- [ ] The favorite button

- [ ] Additional actions

- [ ] The actions of: like, comment, send kudos (article or comment) and share

- [ ] Show the list of kudos and likes

- [ ] The date of the comments
- [ ] The three dots to translate a comment

- [ ] Show all comments

- In News application:

- [ ] The three dots that open actions

- [ ] The button that open the filter

- [ ] The button "Load more"

**Expected behavior:**

Use the native browser focus

![Image](https://github.com/user-attachments/assets/1b41469d-5542-4c47-81bc-455ae2dd5e90)

![Image](https://github.com/user-attachments/assets/a70dcab4-5276-4178-bf08-c45466a124e3)

**Comment by mbensalem** [comment]
Already fixed
